src/Math/babylon.math.ts

```diff
@@ -4564,11 +4564,13 @@
 
         /**
          * Extracts the rotation matrix from the current one and sets it as the given "result"  
+         * @param ignoreScaling when set, do not let negative scale matrix values modify the rotation matrix (Default: false)

```

Comment:
Seems suspicious :)

Reply:
Yea, to me it was odd that scale would effect the rotation matrix. Originally I added this parameter, now I removed it but am setting scale to positive on the mesh prior to calling getRotationMatrix in the gizmo class.